kong/dao/cassandra_db.lua

```diff
@@ -62,7 +62,7 @@ local function serialize_arg(field, value)
     return cassandra.uuid(value)
   elseif field.type == "timestamp" then
     return cassandra.timestamp(value)
-  elseif field.type == "table" then
+  elseif field.type == "table" or field.type == "array" then
```

Comment:
@thefosk For a plugin, what's being deserialized / serialized is the `config` property, which is of "table" type.  See here => https://github.com/Mashape/kong/blob/master/kong/dao/schemas/plugins.lua#L44
That's why it's working for plugins.

So for the acl plugin, blacklist and whitelist properties are not rows that go through this function. To be sure about that, I added the following lines:
```
      if col == "blacklist" then
        rows[i][col] = "hello world"
      end
```
and requested `GET /plugins`. The ACL plugin I configured was correctly rendered without no "hello world".

I've checked all DAO objects. Appart from plugins for which it is the `config` property that is being serialized / deserialized, `oauth2_credentials` really is the first one using an "array" property.